src/app/components/Modals/UnbondWithdrawModal.tsx

```diff
@@ -0,0 +1,84 @@
+import { useRef } from "react";
+import { Modal } from "react-responsive-modal";
+import { IoMdClose } from "react-icons/io";
+
+import { useTheme } from "@/app/hooks/useTheme";
+
+export const UNBOND = "unbond";
+export const WITHDRAW = "withdraw";
```

Comment:
```suggestion
export const MODE_UNBOND = "unbond";
export const MODE_WITHDRAW = "withdraw";
```

to not overlap with other similar names